### PR TITLE
Make "Activate bootloader" button use bootloader in flash if it exists

### PR DESCRIPTION
**Pull request by klutvott123**
There's missing code that causes rebooting into the wrong bootloader when pressing the button.
From the discussion in https://github.com/betaflight/betaflight/issues/11334 it looks like this button should reboot into the bootloader that allows flashing firmware. For targets that have bootloader in flash, this is the one we want.
~~In addition to this I propose we change the behaviour of the `bl` command to use the default bootloader.~~
Edit: Logic in cli code looks like it already behaves as it should

Fixes https://github.com/betaflight/betaflight/issues/11334

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by klutvott123** [comment]
Removed the for discussion label as this is code that should have been there since the bootloader options and the flag in targetCapabilities was added in the firmware

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
